backend/file/data/myshop.trace.db

```diff
@@ -0,0 +1,171 @@
+2022-08-08 07:26:00 jdbc[3]: exception
```

Comment:
Do we need this file for the app to run properly? if not then ignore this for the next PR (pull request)